plugins/luxtronic2/README.md

```diff
@@ -46,6 +56,12 @@ Defines a mapping to a attribute (read-only). All attribute values are bytes (nu
 ### lux2_c
 Defines a mapping to a calculated value (read-only). All calculated values are integer (numbers).
 
+### lux2_unpack
+Python lambda function. Called before the value is sent towards the device.
+
+### lux2_pack
+Called before the value is written to the smarthome item.
+
```

Comment:
@ohinckel I looked a bit further into default values for modbus_pack (same for unpack and lux2). 
I still don't see a good way for `eval_out`. To make something like my last example work, it would have to check the caller e.g. like: `eval_out = return knxVal if caller == 'KNX' else return modbusVal`
I gonna stick with pack(/unpack), if you don't have a better solution.
The somewhat defaults at the moment are `lambda x: x`. I think this means one does not have to use the pack/unpack functions if the type is set to `list`. If *_pack is used it could be any other type and it should be possible to use eval for unpacking. I did not test that.
So again I don't see any better way.